### PR TITLE
Medical Rework

### PR DESCRIPTION
**Pull request by Zargserg**
Moderate rework for medical, focusing on chemicals.

:cl: Zargserg
tweak: Medicines and items tweaked
balance: Balanced basic and advanced tier medication; made limbs weaker
code: Changed several values, check the code for more.
/:cl:

- Changing names to Star Trek counterparts.
- Readding Inaprovaline as a stabilizing medicine.
- Dermaline and Terakine need Kelotane and Bicaridine for production (Replacing Oxygen and Carbon in the recipes)
- Bicaridine and Kelotane can mend moderate injuries but are ineffective for severe and life-threatening injuries. 
- Terakine and Dermaline are effective in treating severe injuries as well as moderate.
- Brute packs and ointment are now only capable of healing minor damage (5pts each).
- Saline-solution does not heal anymore and will compromise breathing if overdosed.

There may be further changes, especially to bleeding and blood-recovery later.

### CODE REVIEW

**Comment by alexkar598** [comment]
The blood related changes will have to be done in another PR for modularization.This PR shall only contain tweaks to chems(brute packs changes are fine)

**Comment by Zargserg** [comment]
Alrighty! I shall focus then on meds, will make an ingame document later for it and then poke KMC.

Right now though I'm on holidays and have no proper rig to work with so stuff has to wait.

**Comment by alexkar598** [comment]
Please list all name changes in your PR body.

**Review by alexkar598 — CHANGES_REQUESTED** [review]
Please list all chem name changes in your PR body

File: code/game/objects/items/stacks/medical.dm [hunk 1]
```diff
@@ -109,7 +109,7 @@
 	icon_state = "brutepack"
 	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
-	heal_brute = 40
+	heal_brute = 10
```
Comment (alexkar598) [inline code/game/objects/items/stacks/medical.dm:112]: Misdocumented change

File: code/game/objects/items/stacks/medical.dm [hunk 2]
```diff
@@ -164,7 +164,7 @@
 	icon_state = "ointment"
 	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
-	heal_burn = 40
+	heal_burn = 10
```
Comment (alexkar598) [inline code/game/objects/items/stacks/medical.dm:167]: Misdocumented change

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 3]
```diff
@@ -217,7 +217,7 @@
 	name = "Spaceacillin"
 	id = "spaceacillin"
 	description = "Spaceacillin will prevent a patient from conventionally spreading any diseases they are currently infected with."
-	color = "#C8A5DC" // rgb: 200, 165, 220
+	color = "#00ffb0" // rgb: 200, 165, 220
```
Comment (alexkar598) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm:220]: Why?

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 4]
```diff
@@ -247,24 +247,24 @@
...
-	name = "Oxandrolone"
-	id = "oxandrolone"
-	description = "Stimulates the healing of severe burns. Extremely rapidly heals severe burns and slowly heals minor ones. Overdose will worsen existing burns."
+/datum/reagent/medicine/dermaline
+	name = "Dermaline"
+	id = "dermaline"
+	description = "Advanced medication that will rapidly rejunivate burn-damaged tissue. Overdose will worsen existing burns."
 	reagent_state = LIQUID
-	color = "#f7ffa5"
+	color = "#eb7327"
```
Comment (alexkar598) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm:255]: Why?

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 5]
```diff
@@ -247,24 +247,24 @@
...
 	metabolization_rate = 0.5 * REAGENTS_METABOLISM
 	overdose_threshold = 25
 
-/datum/reagent/medicine/oxandrolone/on_mob_life(mob/living/carbon/M)
-	if(M.getFireLoss() > 50)
+/datum/reagent/medicine/dermaline/on_mob_life(mob/living/carbon/M)
+	if(M.getFireLoss() > 25)
```
Comment (alexkar598) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm:260]: Document exact number in PR body

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 6]
```diff
@@ -300,7 +300,7 @@
 /datum/reagent/medicine/salglu_solution
 	name = "Saline-Glucose Solution"
 	id = "salglu_solution"
-	description = "Has a 33% chance per metabolism cycle to heal brute and burn damage. Can be used as a temporary blood substitute."
+	description = "Can be used as a temporary blood substitute. Overdosing causes displacement of regular blood oxygen."
```
Comment (alexkar598) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm:303]: Please document that saline still heals blood lost

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 7]
```diff
@@ -486,39 +481,39 @@
...
 	reagent_state = LIQUID
 	color = "#D2D2D2"
 	metabolization_rate = 0.5 * REAGENTS_METABOLISM
 	overdose_threshold = 25
 
 
-/datum/reagent/medicine/sal_acid/on_mob_life(mob/living/carbon/M)
-	if(M.getBruteLoss() > 50)
+/datum/reagent/medicine/terakine/on_mob_life(mob/living/carbon/M)
+	if(M.getBruteLoss() > 25)
```
Comment (alexkar598) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm:495]: Undocumented changed

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 8]
```diff
@@ -542,59 +537,58 @@
...
 
-/datum/reagent/medicine/ephedrine/on_mob_delete(mob/living/L)
+/datum/reagent/medicine/inaprovaline/on_mob_delete(mob/living/L)
 	L.remove_movespeed_modifier(id)
 	..()
 
-/datum/reagent/medicine/ephedrine/on_mob_life(mob/living/carbon/M)
-	M.AdjustAllImmobility(-20, FALSE)
+/datum/reagent/medicine/inaprovaline/on_mob_life(mob/living/carbon/M)
+	M.adjustOxyLoss(-0.5*REM, 0)
```
Comment (alexkar598) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm:558]: Please list all properties of inaprovaline in the PR body

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 9]
```diff
@@ -926,13 +919,17 @@
 /datum/reagent/medicine/bicaridine
 	name = "Bicaridine"
 	id = "bicaridine"
-	description = "Restores bruising. Overdose causes it instead."
+	description = "A basic regenerative substance which can even heal moderate muscular and vascular damage. It has issues handling massive trauma and thus won't heal them effectively."
 	reagent_state = LIQUID
-	color = "#C8A5DC"
+	color = "#fd0000"
```
Comment (alexkar598) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm:924]: Why

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 10]
```diff
@@ -926,13 +919,17 @@
...
 	overdose_threshold = 30
 
+
 /datum/reagent/medicine/bicaridine/on_mob_life(mob/living/carbon/M)
-	M.adjustBruteLoss(-2*REM, 0)
+	if(M.getBruteLoss() > 75)
+		M.adjustBruteLoss(-0.5*REM, 0)
+	else
+		M.adjustBruteLoss(-2*REM, 0)
```
Comment (alexkar598) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm:932]: State exact number in PR body

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 11]
```diff
@@ -978,11 +960,11 @@
 	. = 1
 
 /datum/reagent/medicine/antitoxin
-	name = "Anti-Toxin"
+	name = "Dylovene"
 	id = "antitoxin"
 	description = "Heals toxin damage and removes toxins in the bloodstream. Overdose causes toxin damage."
 	reagent_state = LIQUID
-	color = "#C8A5DC"
+	color = "#05a728"
```
Comment (alexkar598) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm:967]: Why?

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 12]
```diff
@@ -1001,21 +983,21 @@
 /datum/reagent/medicine/inaprovaline
```
Comment (alexkar598) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm:983]: Dual definition of inaprovaline

File: code/modules/reagents/chemistry/recipes/medicine.dm [hunk 13]
```diff
@@ -213,7 +215,7 @@
 	name = "Bicaridine"
 	id = "bicaridine"
 	results = list("bicaridine" = 3)
-	required_reagents = list("carbon" = 1, "oxygen" = 1, "sugar" = 1)
+	required_reagents = list("inaprovaline" = 1, "carbon" = 1)
```
Comment (alexkar598) [inline code/modules/reagents/chemistry/recipes/medicine.dm:218]: Undocumented change

File: code/modules/surgery/bodyparts/bodyparts.dm [hunk 14]
```diff
@@ -224,7 +224,7 @@
 		return BODYPART_DISABLED_PARALYSIS
 	if(can_dismember() && !owner.has_trait(TRAIT_NOLIMBDISABLE))
 		. = disabled //inertia, to avoid limbs healing 0.1 damage and being re-enabled
-		if((get_damage(TRUE) >= max_damage) || (owner.has_trait(TRAIT_EASYLIMBDISABLE) && (get_damage(TRUE) >= (max_damage * 0.6)))) //Easy limb disable disables the limb at 40% health instead of 0%
+		if((get_damage(TRUE) >= max_damage) || (owner.has_trait(TRAIT_EASYLIMBDISABLE) && (get_damage(TRUE) >= (max_damage * 0.45)))) //Easy limb disable disables the limb at 40% health instead of 0%
```
Comment (alexkar598) [inline code/modules/surgery/bodyparts/bodyparts.dm:227]: Undocumented change

File: code/modules/surgery/bodyparts/robot_bodyparts.dm [hunk 15]
```diff
@@ -297,31 +297,31 @@
...
+	max_damage = 15
 
 /obj/item/bodypart/l_leg/robot/surplus
 	name = "surplus prosthetic left leg"
 	desc = "A skeletal, robotic limb. Outdated and fragile, but it's still better than nothing."
 	icon = 'icons/mob/augmentation/surplus_augments.dmi'
 	brute_reduction = 0
 	burn_reduction = 0
-	max_damage = 20
+	max_damage = 15
```
Comment (alexkar598) [inline code/modules/surgery/bodyparts/robot_bodyparts.dm:316]: Undocumented change

**Review by alexkar598 — CHANGES_REQUESTED** [review]

**Comment by alexkar598** [comment]
@Zargserg big dummy me,meant atomization not modularization

**Comment by ben10083-zz** [comment]
is this still being worked on?

**Comment by alexkar598** [comment]
Closing next week if no activity until then

**Comment by Zargserg** [comment]
> Closing next week if no activity until then

Yeh, I simply have lost interest in SS13 at the time being.
Stuff's been a tad too rough and I eventually forgot about the pull-request as well.

**Pull request closed without merging**